operator/cmd/mesh/testdata/manifest-generate/data-snapshot/charts/base/files/gen-istio-cluster.yaml

```diff
@@ -3276,3 +3277,368 @@ spec:
     storage: true
 
 ---
+
+---
+# Source: crds/crd-mixer.yaml
```

Comment:
why we still have the mixer related CRDs here

Reply:
Its just copying from manifests/. They were removed in a later PR, I will update this one again.